### PR TITLE
Fix on class not extending commonobject

### PR DESCRIPTION
**Pull request by atm-maxime**
- return path after task deletion + start of replacing conf->entity by getentity

### CODE REVIEW

File: htdocs/fichinter/class/fichinter.class.php [hunk 1]
```diff
@@ -998,7 +998,7 @@ function fetch_lines()
 /**
  *	Classe permettant la gestion des lignes d'intervention
  */
-class FichinterLigne
+class FichinterLigne extends CommonObject
 {
 	var $db;
 	var $error;
```
Comment (KreizIT) [inline htdocs/fichinter/class/fichinter.class.php:1005]: You should also remove $db,$error and $errors as they are define in CommonObject

**Comment by eldy** [comment]
Objects representing lines of details must extend 
CommonObjectLine instead of CommonObject.

**Comment by atm-maxime** [comment]
Problem is that CommonObjectLine is an empty abstract class and will not contain the "call_trigger" function (which is in commonobject)... And the file containing this class is not even included into pages...

**Comment by eldy** [comment]
Before having a better solution, we could copy the "call_trigger" also into commonobjectline.

Include of the abstract class should not be done by page but by the file that need it. To it must be included into the file that contains the class FichinterLigne.

**Comment by atm-maxime** [comment]
Ok Done

**Pull request merged**